tests/nn/data_parallel/test_fsdp_grad_acc.py

```diff
@@ -17,11 +18,15 @@
 from .test_fsdp import DistributedTest, NestedWrappedModule, rename_test, spawn_and_init
 
 
-class TestNoSync(DistributedTest):
+class TestGradAcc(DistributedTest):
     def test_transformer(self):
         fn = functools.partial(self._test_transformer, config={})
         spawn_and_init(fn)
 
+    def test_transformer_grad_acc_without_no_sync(self):
+        fn = functools.partial(self._test_transformer, config={}, use_no_sync_context=False)
+        spawn_and_init(fn)
```

Comment:
would be cool to assert same loss between the two configs. Since the only tradeoff seems is speed vs. mem.

Reply:
Sure, let me look into it.